lib/pbench/server/database/models/server_settings.py

```diff
@@ -200,92 +200,93 @@ class ServerConfig(Database.Base):
     @staticmethod
     def _default(key: str) -> JSONVALUE:
         try:
-            config = SERVER_CONFIGURATION_OPTIONS[key]
+            setting = SERVER_SETTINGS_OPTIONS[key]
         except KeyError as e:
             if key:
-                raise ServerConfigBadKey(key) from e
+                raise ServerSettingBadKey(key) from e
             else:
-                raise ServerConfigMissingKey() from e
-        return config["default"]()
+                raise ServerSettingMissingKey() from e
+        return setting["default"]()
 
     @staticmethod
     def _validate(key: str, value: JSONVALUE) -> JSONVALUE:
         try:
-            config = SERVER_CONFIGURATION_OPTIONS[key]
+            setting = SERVER_SETTINGS_OPTIONS[key]
         except KeyError as e:
             if key:
-                raise ServerConfigBadKey(key) from e
+                raise ServerSettingBadKey(key) from e
             else:
-                raise ServerConfigMissingKey() from e
-        return config["validator"](key, value)
+                raise ServerSettingMissingKey() from e
+        return setting["validator"](key, value)
 
     @staticmethod
-    def create(key: str, value: JSONVALUE) -> "ServerConfig":
-        """A simple factory method to construct a new ServerConfig setting and
+    def create(key: str, value: JSONVALUE) -> "ServerSetting":
+        """A simple factory method to construct a new ServerSetting setting and
         add it to the database.
 
         Args:
-            key : config setting key
-            value : config setting value
+            key : server setting key
+            value : server setting value
 
         Returns:
-            A new ServerConfig object initialized with the parameters and added
+            A new ServerSetting object initialized with the settings and added
             to the database.
         """
 
         v = __class__._validate(key, value)
-        config = ServerConfig(key=key, value=v)
-        config.add()
-        return config
+        setting = ServerSetting(key=key, value=v)
+        setting.add()
+        return setting
 
     @staticmethod
-    def get(key: str, use_default: bool = True) -> "ServerConfig":
-        """Return a ServerConfig object with the specified configuration key
-        setting.
+    def get(key: str, use_default: bool = True) -> "ServerSetting":
+        """Return a ServerSetting object with the specified key setting.
 
-        For example, ServerConfig.get("dataset-lifetime").
+        For example, ServerSetting.get("dataset-lifetime").
 
         If the setting has no definition, a default value will optionally
         be provided.
 
         Args:
-            key : System configuration setting name
+            key : System setting name
```

Comment:
You've been replacing "system" and/or "configuration" with "server"; should we do that here, as well?

Reply:
Yes ... I think.